src/counterfactual_mean_based/estimators.jl

```diff
@@ -117,6 +125,8 @@ function (tmle::Tmle)(Ψ::StatisticalCMCompositeEstimand, dataset; cache=Dict(),
     # Get propensity score truncation threshold
     n = nrows(nomissing_dataset)
     ps_lowerbound = ps_lower_bound(n, tmle.ps_lowerbound)
+    # Get prevalence weights for CCW-TMLE step 
+    prevalence_weights = get_weights_from_prevalence(tmle.prevalence, nomissing_dataset[!, relevant_factors.outcome_mean.outcome])
```

Comment:
If I understand correctly, the prevalence weights only depend on the user provided q_0 and the number of controls J, so these weights could be computed only once at the begining of the procedure

Reply:
No you are correct, they could be computed only once be we'd be left with the difficult problem of assigning them to each individual in the original or nomissing dataset!